src/chapters/thymeleaf-views/template-fragments.rst

```diff
@@ -0,0 +1,204 @@
+Template Fragments
+====================
+
+Just like methods in Java provide us with the ability to reuse useful code,
+Thymeleaf allows us to do something similar with HTML.
+
+.. index:: ! fragments
+
+**Fragments** are blocks of HTML elements that we want to use across multiple
+templates. The fragments are stored in a separate file, and they can be
+accessed by using the keywords ``fragment`` and ``replace``.
+
+Try It!
+--------
+
+Code along with the following video to practice using fragments in your
+templates:
+
+.. todo::
+
+   ``Coding Events App: Refactor Templates to Use Fragments`` video goes
+   here...
+
+A summary of ``fragment`` and ``replace`` it given below, but remember that the
+text supports the video and is NOT intended as a replacement.
```

Comment:
I really like the "Try It!", but it is difficult to understand the video without the conceptual context coming first.

Reply:
OK.  I moved the video to the bottom of the page.